mne/viz/tests/test_3d.py

```diff
@@ -318,4 +324,30 @@ def test_snapshot_brain_montage():
     assert_raises(TypeError, snapshot_brain_montage, fig, info)
 
 
+@requires_nilearn
+@spm_face.requires_spm_data
```

Comment:
we can just take `lh.ribbon.mgz` and `rh.ribbon.mgz` from the sample data and add them to the testing data, they are ~150 kB apiece